content/applications/finance/documents.rst

```diff
@@ -120,4 +208,14 @@ Select the *Actions* tab to set up your action. You can simultaneously:
 
 .. image:: documents/workflow-action-example.png
    :align: center
-   :alt: Example of a workflow action Odoo Documents
\ No newline at end of file
+   :alt: Example of a workflow action Odoo Documents
+
+Digitize documents with AI and optical character recognition (OCR)
+==================================================================
+
+Documents available in the Finance workspace can be digitized. Select the document you want to
+digitize, click on :guilabel:`Create Bill`, :guilabel:`Create Customer Invoice` or
+:guilabel:`Create credit note`, and then click on :guilabel:`Send for Digitization`.
+
+.. seealso::
+   - `AI-powered document digitization <https://www.odoo.com/documentation/15.0/applications/finance/accounting/payables/supplier_bills/invoice_digitization.html>_
```

Comment:
```suggestion
   :doc:`AI-powered document digitization <../accounting/payables/supplier_bills/invoice_digitization>
```
[You should link to other docs this way](https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#documentation-pages-hyperlinks). 
However, I assume the digitization page wasn't merged yet I assume, so this link won't work either in any case.